db/migrate/20170825090802_add_minutes_to_hours.rb

```diff
@@ -0,0 +1,6 @@
+class AddMinutesToHours < ActiveRecord::Migration[5.1]
+  def change
+    rename_column :hours, :duration, :hours
+    add_column :hours, :minutes, :integer
+  end
+end
```

Comment:
Just a very minor point: if you're introducing a new table in a PR I think it's fine to just update the original migration, instead of adding more migrations to change the table.

Reply:
Just a side Question, generally to the migrations (not related to this PR): 

Is there a decent (proper) way to squash all those migrations to have it more overseeable?

Reply:
There's the [squasher gem](https://github.com/jalkoby/squasher) but I never used it.

Basically you can take the contents of `db/schema.rb` and use it as a single migration, [GitLab did this once](https://gitlab.com/gitlab-org/gitlab-ce/commit/e8f8abddb0).

Reply:
I'll bring this up, so the team and seb can decide if something like that or other ways of slickening the migrations should be done... thx.

Reply:
I wouldn't worry about it until we get to several hundred migrations :)